src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -928,6 +929,62 @@ public HyperwalletWebhookNotification getWebhookEvent(String webhookToken) {
     }
 
     //--------------------------------------
+    // Transfer Methods
+    //--------------------------------------
+
+    /**
+     * Create a Transfer Method
+     *
+     * @param jsonCacheToken String JSON cache token
+     * @param transferMethod TransferMethod object to create
+     * @return HyperwalletTransferMethod Transfer Method object created
+     */
+    public HyperwalletTransferMethod createTransferMethod(String jsonCacheToken, HyperwalletTransferMethod transferMethod) {
+
+        if (transferMethod == null || StringUtils.isEmpty(transferMethod.getUserToken())) {
+            throw new HyperwalletException("User token is required");
+        }
+        if (StringUtils.isEmpty(jsonCacheToken)) {
+            throw new HyperwalletException("JSON token is required");
+        }
+        transferMethod = copy(transferMethod);
```

Comment:
according to the documentation i was following you are correct. 
https://confluence.site1.hyperwallet.local/pages/viewpage.action?pageId=22299508

the other SDKs seem to handle it at the service level.